resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketForm/GameHashSelectField.tsx

```diff
@@ -0,0 +1,81 @@
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import type { FC } from 'react';
+import { useFormContext } from 'react-hook-form';
+
+import {
+  BaseFormControl,
+  BaseFormDescription,
+  BaseFormField,
+  BaseFormItem,
+  BaseFormLabel,
+} from '@/common/components/+vendor/BaseForm';
+import {
+  BaseSelect,
+  BaseSelectContent,
+  BaseSelectItem,
+  BaseSelectTrigger,
+  BaseSelectValue,
+} from '@/common/components/+vendor/BaseSelect';
+import { usePageProps } from '@/common/hooks/usePageProps';
+
+import type { CreateAchievementTicketFormValues } from './useCreateAchievementTicketForm';
+
+export const GameHashSelectField: FC = () => {
+  const { gameHashes } = usePageProps<App.Platform.Data.CreateAchievementTicketPageProps>();
+
+  const { t } = useLaravelReactI18n();
+
+  const form = useFormContext<CreateAchievementTicketFormValues>();
+
+  const namedHashes = gameHashes.filter((hash) => !!hash.name?.trim());
+  const unnamedHashes = gameHashes.filter((hash) => !hash.name?.trim());
+
+  const sortedNamedHashes = [...namedHashes].sort((a, b) => a.name!.localeCompare(b.name!));
+  const sortedUnnamedHashes = [...unnamedHashes].sort((a, b) => a.md5.localeCompare(b.md5));
+
+  const sortedHashes = [...sortedNamedHashes, ...sortedUnnamedHashes];
+
+  return (
+    <BaseFormField
+      control={form.control}
+      name="hash"
+      render={({ field }) => (
+        <BaseFormItem className="flex w-full flex-col gap-1 sm:flex-row sm:items-center">
+          <BaseFormLabel
+            htmlFor="hash-select"
+            className="text-menu-link sm:mt-[13px] sm:min-w-36 sm:self-start"
+          >
+            {t('Supported Game File')}
+          </BaseFormLabel>
+
+          <div className="flex w-full flex-col gap-1">
+            <BaseFormControl>
+              <BaseSelect
+                onValueChange={field.onChange}
+                defaultValue={field.value ? String(field.value) : undefined}
+              >
```

Comment:
Is there some way to make this stretch full? It's weird to see it wrap when there's clearly more room on the right:
![image](https://github.com/user-attachments/assets/4fbf9187-364d-4d45-ae03-de583c12b304)

Reply:
Not a problem - made this change in latest.